content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -1,47 +1,66 @@
-===========================
-Manage multiple sales teams
-===========================
+====================
+Multiple sales teams
+====================
 
-In Odoo, you can manage several sales teams, departments or channels
-with specific sales processes. To do so, we use the concept of *Sales
-Channel*.
+In Odoo, you can manage several sales teams, departments, or channels, each with their own unique
+sales processes, using *Sales Teams*.
 
-Create a new sales channel
-==========================
 
-To create a new *Sales Channel*, go to :menuselection:`Configuration --> Sales Channels`.
+Create a new Sales Team
+=======================
 
-There you can set an email alias to it. Every message sent to that email
-address will create a lead/opportunity.
+To create a new *Sales Team*, go to :menuselection:`CRM --> Configuration --> Sales Teams` then
+click **Create**.
 
-.. image:: media/multi_sales_team01.png
-    :align: center
+On the creation page, you can set an *Email Alias* for your team. Every message sent to that email
+address will create a lead/opportunity in your pipeline. You can also choose whether you
+accept emails from *Everyone*, *Authenticated Partners*, or *Followers Only*.
 
-Add members to your sales channel
----------------------------------
+You can set up a *Domain* if you want to assign leads/opportunities to your team based on specific
+filters, such as country, language, etc. Set an *Invoicing Target* if your team has specific
+monthly revenue targets they need to achieve.
```

Comment:
Removing "you" and improving wording

`Set a *Domain* to assign leads/opportunities to this Sales Team based on specific filters, such as country, language, or campaign. Set an *Invoicing Target* if this team has specific monthly revenue goals.`